### DISCUSSION TITLE
Support for FoxESS H3 Pro Inverter

### DISCUSSION THREAD

**Discussion by lauricthepilot** [StealthChesnut/HA-FoxESS-Modbus#132] [Q&A]
Hello. 

Is there any reason why this integration would not work for a FoxESS H3 Pro Inverter (the 3-phase big Brother of the H1 series)? I see that the H3 is not specifically listed, although the H3 is a very new inverter, and the last contributions here were made a year ago. Does anyone know why it might not work?

Here is a link to the H3 Series: https://www.fox-ess.com/3-phase/

Cheers

**Comment by Daibutt** [comment]
Might be worth swapping to this integration that is maintained;

https://github.com/nathanmarlor/foxess_modbus

It says it works with H3, not sure about Pro.

**Comment by lauricthepilot** [comment]
Ah, Brilliant. Thank you
